pom.xml

```diff
@@ -216,7 +216,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-resources-plugin</artifactId>
-                <version>3.2.0</version>
+                <version>3.1.0</version>
```

Comment:
How do we stop Dependabot from complaining about this vulnerable version?

Reply:
We can add exclusions by adding an ignored_updates block to the .github/dependabot.yml file, haven't done this yet as i'm still investigating the upgrade to unblock this, just reverting as a temporary measure to get builds working again while i track down the build code for the container.